### ISSUE TITLE
Site seems down?

### ISSUE THREAD

**Issue by hrxn** [nhdez/hn-blogroll#1]
Hey, the site has been down for me the last weeks, apparently..

Is this just a maintenance break? Geofencing issue? Or really gone for good?

**Comment by nhdez** [comment]
Yeah, sorry about that, it's been a while. Domain name expired, totally missed that renewal. I think I am going to try to bring this back up soon, but under a different domain name. I'll update the README accordingly.
